node-directory-sync-example/README.md

```diff
@@ -9,38 +9,40 @@ Node.js version 10+
 ## Node Project Setup
 
 1. Clone the main repo and install dependencies for the app you'd like to use:
+
     ```bash
     # HTTPS
-    git clone https://github.com/workos-inc/node-example-applications.git
+    git clone https://github.com/workos/node-example-applications.git
     ```
+
     or
 
     ```bash
     # SSH
-    git clone git@github.com:workos-inc/node-example-applications.git
+    git clone git@github.com:workos/node-example-applications.git
     ```
 
-2. Navigate to Directory Sync app within the cloned repo. 
-   ```bash
-   $ cd node-example-applications/node-directory-sync-example
-   ```
+2. Navigate to Directory Sync app within the cloned repo.
 
-3. Install the dependencies. 
     ```bash
-    $ npm install
+    $ cd node-example-applications/node-directory-sync-example
     ```
 
+3. Install the dependencies.
+    ```bash
+    $ npm install
+    ```
 
 ## Configure your environment
 
 1. Grab your [API Key](https://dashboard.workos.com/api-keys).
 2. Get your [Client ID](https://dashboard.workos.com/sso/configuration).
```

Comment:
Client ID actually isn't needed to run the Dsync App. We should probably remove it from the ReadME since it's not being used. I'll remove it from the Dsync apps for Python and Typescript too.